### PR TITLE
Store various values of heart rate 

### PR DESCRIPTION
**Pull request by Priyanshu078**
Added code to add the heart rate values in a text file every time when the user measures the heart rate successfully.

Check Heart Rate screen
homeScreen

When the user has not measured his/her heart rate from the app
nodata

When the heartRate is measured sometimes by the user
heartRate values

### CODE REVIEW

File: lib/homePage.dart [hunk 1]
```diff
@@ -188,27 +194,48 @@ class HomePageView extends State<HomePage>
...
+                        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                        children: [
+                          Transform.scale(
+                            scale: _buttonScale,
+                            // child: Row(
+                            //     mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                            //     children: <Widget>[
+                            child: MaterialButton(
+                              shape: RoundedRectangleBorder(
+                                  borderRadius: BorderRadius.circular(30)),
```
Comment (anushbhatia) [inline lib/homePage.dart:203]: keep the border-radius to be low so that they look more like cards rather than buttons

File: lib/homePage.dart [hunk 2]
```diff
@@ -1,10 +1,14 @@
 import 'dart:async';
+// import 'dart:html';
```
Comment (anushbhatia) [inline lib/homePage.dart]: Don't comment remove this line completely.

File: lib/homePage.dart [hunk 3]
```diff
@@ -188,27 +194,48 @@ class HomePageView extends State<HomePage>
...
-                          }
-                        },
-                      ),
-                    ),
+                    Row(
+                        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                        children: [
+                          Transform.scale(
+                            scale: _buttonScale,
+                            // child: Row(
```
Comment (anushbhatia) [inline lib/homePage.dart]: here too remove if not required

**Review by anushbhatia — CHANGES_REQUESTED** [review]

**Comment by Priyanshu078** [comment]
Decreased Border Radius
BPM

**Comment by anushbhatia** [comment]
Resolve the conflict rest it is ready to be merged.

**Pull request merged**